src/main/java/com/microsoft/sqlserver/jdbc/SQLServerDriver.java

```diff
@@ -563,7 +563,7 @@ static PrepareMethod valueOfString(String value) throws SQLServerException {
 
 enum SQLServerDriverStringProperty {
     APPLICATION_INTENT("applicationIntent", ApplicationIntent.READ_WRITE.toString()),
-    APPLICATION_NAME("applicationName", SQLServerDriver.DEFAULT_APP_NAME),
```

Comment:
Can we keep this unchanged to default to DEFAULT_APP_NAME

Reply:
Sure, Its not needed